MadDonkeySoftware/RAG-Report
# Initial commit of template.

### Story
Fixes #4 

### Summary 
Built an initial template for the RAG report project

### Testing
* Navigate to http://127.0.0.1:8000 and verify a 404 page is displayed
* Navigate to http://127.0.0.1:8000/users/me redirects you to login page
